guides/Development/hack_on_wanda.adoc

```diff
@@ -5,21 +5,21 @@
 In order to run Wanda, the following software must be installed:
 
 [arabic]
-. https://elixir-lang.org/[Elixir]
-. https://www.erlang.org/[Erlang OTP]
-. https://www.rust-lang.org/tools/install[Rust]
-. https://docs.docker.com/get-docker/[Docker]
-. https://docs.docker.com/compose/install/[Docker Compose]
+. link:https://elixir-lang.org/[Elixir]
```

Comment:
Reason why i add a link everywhere is i want to have a ci script which ensures the link's are still valid in the future